torch/csrc/autograd/functions/convolution.cpp

```diff
@@ -55,7 +55,7 @@ auto ConvParams::is_output_padding_neg() const -> bool {
 
 auto ConvParams::is_output_padding_big() const -> bool {
   bool is_big = false;
-  for (int i = 0; i < output_padding.size(); i++) {
+  for (size_t i = 0; i < output_padding.size(); i++) {
```

Comment:
I know it fixes a warning but it will conflict with upstream that has it fixed already